ramls/types/resources/resourcePutDataAttributes.json

```diff
@@ -0,0 +1,110 @@
+{
```

Comment:
We might want to double check and verify from UI - folio.frontside.io what it is passing as payload for a PUT request to /eholdings/resources - If it is passing the complete payload as a GET request, then we might not need a separate `resourcePutDataAttributes.json` schema, we could re-use the `resourceDataAttributes.json` instead.

Reply:
Unfortunately, we can't do that, because in that case we will have the ability to edit some properties that shouldn't be edited by put request, for example "isTitleCustom", "titleId", "subjects". I checked https://s3.amazonaws.com/foliodocs/api/mod-kb-ebsco/eholdings.html# to be sure that there shouldn't be such properties in put request.

Reply:
I took a look at payload for PUT request from UI

`PUT https://okapi.frontside.io/eholdings/resources/18-34-351`

As you can see from below we are getting all attributes (although some of these properties -- as noted above are read only). I ran into this same issue with Provider PUT.  The below payload will fail if the resourcePutDataAttributes has only the fields that can be edited  - unfortunately it needs the complete resourceDataAttributes (some fields are read only)

```
{"data":{"id":"18-34-351","type":"resources","attributes":{"description":"A magazine for lawyers published by the American Bar Association.  Articles on substantive law, practice tips and lifestyle for the practicing lawyer.","edition":"","isPeerReviewed":false,"isTitleCustom":false,"publisherName":"American Bar Association","titleId":351,"contributors":[],"identifiers":[{"id":"0747-0088","type":"ISSN","subtype":"Print"},{"id":"10269097","type":"TSDID","subtype":"Empty"},{"id":"2162-7983","type":"ISSN","subtype":"Online"},{"id":"351","type":"BHM","subtype":"Empty"},{"id":"ABJ","type":"Mid","subtype":"Empty"}],"name":"ABA Journal","publicationType":"Journal","subjects":[{"type":"TLI","subject":"Law"},{"type":"TLI","subject":"Legal Profession"}],"coverageStatement":"","customEmbargoPeriod":{"embargoValue":0,"embargoUnit":null},"isSelected":true,"managedEmbargoPeriod":{"embargoUnit":"days","embargoValue":14},"packageId":"18-34","packageName":"Academic ASAP","providerId":18,"providerName":"Gale | Cengage","visibilityData":{"isHidden":true,"reason":""},"managedCoverages":[{"beginCoverage":"2007-05-01","endCoverage":""}],"customCoverages":[],"proxy":{"id":"<n>","inherited":true}}}}
```

Reply:
thanks, updated